crates/kilt-dip-primitives/src/state_proofs/substrate_no_std_port.rs

```diff
@@ -0,0 +1,69 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2023 BOTLabs GmbH
```

Comment:
if this code is copied from somewhere, do we need to include a copyright notice from the source?

Reply:
https://github.com/KILTprotocol/kilt-node/pull/602/commits/e20c96d21fb38dc1a812dac90dd7c1b2b2db70d0

Reply:
Do you think this is good now?